src/EduManagementLab.Web/Program.cs

```diff
@@ -17,6 +17,12 @@
 builder.Services.AddRazorPages();
 
 var app = builder.Build();
+using (var scope = app.Services.CreateScope())
```

Comment:
Dont seed in startup. We can use the test Database.DropCreateAndSeed_EduManagementLabDb to drop, create and seed when we want to reseed the database.

Reply:
Noted. I've updated the file 👍🏼